perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -679,15 +679,15 @@ sub setup_categories_and_bodies : Private {
 
             $non_public_categories{ $contact->category } = 1 if $contact->non_public;
         }
-        $seen{$contact->category} = $contact->category_display;
+        $seen{$contact->category} = $contact;
     }
 
     if (@category_options) {
         # If there's an Other category present, put it at the bottom
         @category_options = (
-            { name => _('-- Pick a category --'), value => _('-- Pick a category --'), group => '' },
-            grep { $_->{name} ne _('Other') } @category_options );
-        push @category_options, { name => _('Other'), value => $seen{_('Other')}, group => _('Other') } if $seen{_('Other')};
+            { category => _('-- Pick a category --'), category_display => _('-- Pick a category --'), group => '' },
+            grep { $_->category ne _('Other') } @category_options );
+        push @category_options, $seen{_('Other')} if $seen{_('Other')};
     }
 
     $c->cobrand->call_hook(munge_category_list => \@category_options, \@contacts, \%category_extras);
```

Comment:
Just a note that anywhere that uses this hook will need updated although I can't find anywhere that does.